src/main/java/core/basesyntax/MainFruitsService.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Apple;
+import core.basesyntax.model.Banana;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Orange;
+import core.basesyntax.service.FileReaderService;
+import core.basesyntax.service.FileWriterService;
+import core.basesyntax.service.StorageService;
+import core.basesyntax.service.TransactionLog;
+import core.basesyntax.service.impl.FileReaderServiceImpl;
+import core.basesyntax.service.impl.FileWriterServiceImpl;
+import core.basesyntax.service.impl.LogIteratorServiceImpl;
+import core.basesyntax.service.impl.StorageAdditionServiceImpl;
+import core.basesyntax.service.impl.StorageReductionServiceImpl;
+import core.basesyntax.strategy.TransDistrStrategy;
+import core.basesyntax.strategy.TransDistrStrategyImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class MainFruitsService {
+    static final String READ_DATA_PATH = "src/main/resources/input.csv";
+    static final String WRITE_REPORT_PATH = "src/main/resources/report.csv";
+    static final String PURCHASE = "p";
+    static final String RETURN = "r";
+    static final String SUPPLY = "s";
+    static final String BALANCE = "b";
+
+    public static void main(String[] args) {
+        Fruit apple = new Apple("apple");
+        Fruit banana = new Banana("banana");
+        Fruit orange = new Orange("orange");
```

Comment:
I don't know if this is correct, but what will you do if you have to add a new fruit. You will have to create a new class??